### ISSUE TITLE
Implement Flush Feature

### ISSUE THREAD

**Issue by dvonthenen** [deepgram/deepgram-dotnet-sdk#250]
## Proposed changes

You don't need to wait for this to be implemented in the SDK. You can use this right now. You can send the following message in the `send()` function:
```
{ "type": "Finalize" }
```

**Comment by tomkail** [comment]
For others who just want the feature and don't really understand the details of implementation, here's some code and how you might use it:

Our use case is this:
- Send microphone data each frame.
- Once the user presses Stop, stop sending data and Flush deepgram.
- Wait for isFinal event from Deepgram (ensuring we have text for the entire time)
- Stop Deepgram

I've implemented Flush like this:

```byte[] byteArray = Encoding.UTF8.GetBytes("{ \"type\": \"Finalize\" }\n");
            deepgramLive.SendMessage(byteArray);```

**Comment by dvonthenen** [comment]
Will be available in the 4.1 release.
